libs/x-content/build.gradle

```diff
@@ -6,26 +6,39 @@
  * Side Public License, v 1.
  */
 
+
+import org.elasticsearch.gradle.transform.UnzipTransform
+import org.gradle.api.internal.artifacts.ArtifactAttributes
+
+import java.util.stream.Collectors
+
 apply plugin: 'elasticsearch.build'
 apply plugin: 'elasticsearch.publish'
 
+configurations {
```

Comment:
If we plan on using this pattern elsewhere it's probably worth placing this into a proper binary plugin with strongly typed task implementations and test coverage.

Reply:
That will be the first thing we do once we are ready to reuse the model for eg logging.